garage/tf/algos/npo.py

```diff
@@ -17,18 +17,19 @@
 from garage.tf.misc.tensor_utils import flatten_batch_dict
 from garage.tf.misc.tensor_utils import flatten_inputs
 from garage.tf.misc.tensor_utils import graph_inputs
-from garage.tf.optimizers import LbfgsOptimizer
+from garage.tf.optimizers import PenaltyLbfgsOptimizer
 
 
 @unique
 class PGLoss(Enum):
     VANILLA = "vanilla"
-    CLIP = "clip"
+    SURROGATE_VANILLA = "surrogate_vanilla"
+    SURROGATE_CLIP = "surrogate_clip"
 
 
 class NPO(BatchPolopt):
     def __init__(self,
-                 pg_loss=PGLoss.VANILLA,
+                 pg_loss=PGLoss.SURROGATE_VANILLA,
                  clip_range=0.01,
```

Comment:
while we are here, i think we should separate the `clip_range` and `step_size` parameters, since they are different numbers in the literature.

`step_size` refers to the maximum KL divergence between old and new policies, as in TRPO
`clip_range` refers to the limit on the likelihood ratio between policies, as in PPO.

Reply:
Is the `step_size` same as the `epsilon` in f(kl) < epsilon?

Reply:
I would then explicitly rename them for what they are `kl_max_step` , `loss_clip_range`